### PR TITLE
Add test verifying resizing of canvas's back buffer with preserveDraw…

### PR DESCRIPTION
**Pull request by kenrussell**
…ingBuffer=true.

The preserveDrawingBuffer=false case is exercised by several other tests. This test doesn't catch any new bugs, but is intended as verification for http://crbug.com/557848 .

### CODE REVIEW

**Comment by kenrussell** [comment]
Please review. Verified with Chrome Canary and Firefox Nightly on Mac OS X.

**Comment by kenrussell** [comment]
(CC: @zhenyao @toji ; FYI @ds-hwang )

**Comment by RafaelCintron** [comment]
I verified in Edge and IE11.  LGTM.

File: sdk/tests/conformance/canvas/canvas-preservedrawingbuffer-resize-test.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+  wtu.setupUnitQuad(gl);
+
+  wtu.requestAnimFrame(render);
+}
+
+function render() {
+  var resultGreen = false;
+
+  if (currentSize == largeSize) {
```
Comment (toji) [inline sdk/tests/conformance/canvas/canvas-preservedrawingbuffer-resize-test.html]: If the point of this test is to catch cases in Chrome where an immutable texture is accidentally recycled and we attempt to resize it, I don't know that this ping-ponging between a single large an small size will predictably catch it. Depending on the recycling behavior you may just happen to click into a stride that recycles the large and small buffers at the right time by accident. I think the test would be more effective if there were more variants or a more difficult-to-hit pattern involved.

Comment (kenrussell) [inline sdk/tests/conformance/canvas/canvas-preservedrawingbuffer-resize-test.html]: Thanks, will update this to alternate sizes every few frames instead.

**Comment by kenrussell** [comment]
@toji Please re-review. Thanks.

**Comment by toji** [comment]
LGTM

**Comment by ds-hwang** [comment]
Thank you for adding this test! I verified this in Ubuntu with and without http://crrev.com/1449043005
LGTM with comments

```The preserveDrawingBuffer=false case is exercised by several other tests. ```
WebglConformance.conformance_textures_webgl_canvas_tex_image_and_sub_image_2d_with_webgl_canvas covers ```preserveDrawingBuffer=false``` case, but it's not intended. It's just by change. IMO, it's better for this test to cover both ```preserveDrawingBuffer=true``` and ```preserveDrawingBuffer=false``` cases

**Comment by kenrussell** [comment]
@ds-hwang without the style: attribute the text jumps all over the place, which is jarring.

When generalizing this test it really became apparent it's a duplicate of the existing rapid-resizing.html test so I'm going to submit a pull request that deletes this new test and generalizes the previous one.

**Pull request merged**